daprdocs/content/en/reference/components-reference/supported-pubsub/setup-pulsar.md

```diff
@@ -80,6 +81,14 @@ spec:
 | publicKey          | N  | A public key to be used for publisher and consumer encryption. Value can be one of two options: file path for a local PEM cert, or the cert data string value  |
 | privateKey          | N  | A private key to be used for consumer encryption. Value can be one of two options: file path for a local PEM cert, or the cert data string value  |
 | keys          | N  | A comma delimited string containing names of [Pulsar session keys](https://pulsar.apache.org/docs/3.0.x/security-encryption/#how-it-works-in-pulsar). Used in conjunction with `publicKey` for publisher encryption |
+| authType | N | One of `"none"`, `"token"` or `"oidc"`. The type of authentication mechanism to use. Defaults to `"none"`, unless `token` is defined in which case it defaults to `"token"`. | "oidc" |
+| token | N  | `authType: token`. Token used for authentication.  | [How to create pulsar token](https://pulsar.apache.org/docs/en/security-jwt/#generate-tokens)|
```

Comment:
Given the setting required for oidc I suggest that there is a separate example of how these are set in the meta. For example this component https://docs.dapr.io/reference/components-reference/supported-pubsub/setup-azure-eventhubs/#azure-active-directory-aad-authentication shows how to set AAD authentication correctly. 

And if there are any particular gochas outside of the the description and examples, add this into a paragraph or two. In otherwords be more specific to user on how to use the oidc settings

Reply:
Makes sense, will do :+1: